cmd/webhook/main.go

```diff
@@ -79,7 +79,12 @@ func main() {
 	configMapWatcher.Watch(channeldefaulter.ConfigMapName, channelDefaulter.UpdateConfigMap)
 
 	if err = configMapWatcher.Start(stopCh); err != nil {
-		logger.Fatalf("failed to start webhook configmap watcher: %v", err)
+		logger.Fatalf("failed to start webhook configmap watcher: %v", zap.Error(err))
+	}
+
+	stats, err := webhook.NewStatsReporter()
+	if err != nil {
+		logger.Fatalw("failed to initialize the stats reporter", zap.Error(err))
```

Comment:
silly but I think it would be great to capitalize `f`ailed

Reply:
it turns into an error and errors should be lowercase.

Reply:
good to know!